Package.swift

```diff
@@ -1,10 +1,10 @@
-// swift-tools-version:5.2
+// swift-tools-version:6.0
 import PackageDescription
 
 let package = Package(
     name: "Imperial",
     platforms: [
-       .macOS(.v10_15)
+       .macOS(.v14)
```

Comment:
It would be worth clarifying the reason for targeting `.macOS(.v14)` instead of the minimum required `.macOS(.v13)` by JWTKit. Is there a specific feature or improvement in `.v14` that we depend on, or could we revert to ensure broader compatibility?

Reply:
No not really, I was planning ahead for a future update to JWTKit v5 and mistakenly thought it required macOS 14